### PR TITLE
backend: transition to @safe code part 2

### PR DESCRIPTION
**Pull request by WalterBright**
Marking everything @trusted that fails to compile with @safe.

### CODE REVIEW

**Comment by UplinkCoder** [comment]
wouldn't this be considered whitewashing?

**Review by maxhaton — APPROVED** [review]

**Comment by WalterBright** [comment]
> wouldn't this be considered whitewashing?

See the rationale here https://github.com/dlang/dmd/pull/12327

File: src/dmd/backend/divcoeff.d [hunk 1]
```diff
@@ -253,6 +255,7 @@ extern (C) bool udiv_coefficients(int N, ullong d, int *pshpre, ullong *pm, int
     return mhighbit;
 }
 
+@trusted
```
Comment (Geod24) [inline src/dmd/backend/divcoeff.d:258]: Why would you need that on a `unittest` ?

Comment (WalterBright) [reply src/dmd/backend/divcoeff.d:258]: Because that unittest does unsafe things and triggers the compiler.

Comment (WalterBright) [reply src/dmd/backend/divcoeff.d:258]: Note that in all this I am **not** attempting to fix **any** of the unsafe code. I am instead just marking everything as @safe, and then marking as @trusted anything that fails to compile. This is what I've recommended before for anyone wanting to transition to safe-by-default code. It's high time we dogfooded this.

**Review by Geod24 — APPROVED** [review]

**Pull request merged**